### ISSUE TITLE
Maven Environment Variable not exporting anymore

### ISSUE THREAD

**Issue by abser** [Microsoft/vscode-maven#250]
I have added my Java Project environment variables in the settings.json. It was working fine. Once I run maven clean, new terminal window open and export my env variables. Since the last 2 days, the environment variables are not exporting to a new terminal anymore. I tried to closing and opening it again, still no luck.

Appreciate your suggestion or fix on it.

**Comment by Eskibear** [comment]
A change related with custom env was shipped with v0.14.2. Previously it explicitly executes `export foo=bar` to inject custom envs, now the envs are injected silently on creating the terminal. 

I just tried in both Window 10 and Mac, and it works well. Could you please have a check again to see if the envs are properly set?

**Comment by abser** [comment]
Hi @Eskibear ,
Yes.  Now I got it working. The env vars are silently exported. I got the problem because of my below environment variable value was an integer.

```
{
      "environmentVariable": "APPLICATION_PORT",
      "value": 51051
}
```
after changing it to String it exporting to console.

```
{
      "environmentVariable": "APPLICATION_PORT",
      "value": "51051"
}
```
